README.md

```diff
@@ -26,7 +33,60 @@ struct Person {
 }
 
 fn main() {
-    let conn = Connection::connect("postgres://postgres@localhost", SslMode::None).unwrap();
```

Comment:
I was under the impression that unwrap is generally discouraged

On Aug 14, 2016 4:43 PM, "Steven Fackler" <notifications@github.com> wrote:

> In README.md
> <https://github.com/sfackler/rust-postgres/pull/194#discussion_r74711071>:
>
> > @@ -26,7 +33,60 @@ struct Person {
> >  }
> >
> >  fn main() {
> > -    let conn = Connection::connect("postgres://postgres@localhost", SslMode::None).unwrap();
>
> Any particular reason to change this?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sfackler/rust-postgres/pull/194/files/d5bfc161a799f5044c6dd93df8a394d43ffcb828#r74711071>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABMFFlqLQyGXHibLK4D9RdMMKQP351cZks5qf6gugaJpZM4JeYP7>
> .
>